src/header.ts

```diff
@@ -0,0 +1,29 @@
+import { PanelLayout, Widget } from '@phosphor/widgets';
+
+import { DebuggerService } from './service';
+
+export class SidebarHeader extends Widget {
```

Comment:
Let's try to add docstrings whenever we add new classes / methods?

Reply:
cf #139

Reply:
The structure of the docstring will be similar to the one used for the other classes.